test/tests/parser/parse_vector_realvectorvalue/parse_vectorvalue.i

```diff
@@ -0,0 +1,22 @@
+[Problem]
+  solve = false
+[]
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 10
+  ny = 10
+[]
+
+[UserObjects]
+  [parsee_tester]
+    type = ReadVectorValue
+    vector_realvv = '0.1 0.2 0.3 0.4 0.5 0.6'
+    vector_point = '0.1 0.2 0.3 0.4 0.5 0.6'
```

Comment:
unused

Reply:
if unused the test requirement must be amended then

Reply:
Got it thanks guys!